CHANGELOG.md

```diff
@@ -62,6 +62,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- **CUMULUS-2371**
+  - Added `archiveMessage` to `sqs-message-consumer` which archives all incoming SQS messages to S3.
+  - Added `deleteArchivedMessage` to `sqs-message-remover` which deletes archived SQS message from S3 once it has been processed.
```

Comment:
These are no longer quite accurate. We should note the behavior change in each lambda, but these functions were not added to those Lambdas

Reply:
[`c8bed0a` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/c8bed0a333136af6c382568ed4831b83d0b662ef)
Apologies, I updated this in two different commits